### DISCUSSION TITLE
Adding a repository on Git hub for android

### DISCUSSION THREAD

**Discussion by cheese-lover** [community/community#45517] [New to GitHub]
Hey Fam how do I add a repository on my github for android

**Comment by jge162** [comment]
@cheese-lover Go to GitHub and log in to your account on your Android phone.

Click on the "+" icon in the top right corner of the page, and then select "New repository".

Give your repository a name, and add a brief description if you like.

Select "Create repository".

Done.

**Reply by cheese-lover** [reply]
Thanks I will come back again if i fail to locate it

**Reply by cheese-lover** [reply]
![Screenshot_20230127_130014_com.github.android.jpg](https://user-images.githubusercontent.com/119293366/215070836-0e729f71-5929-42a6-9274-f7d9e20e8388.jpg)
It is giving me an option of creating an issue not a repository 😭

**Reply by jge162** [reply]
glad to help. should be that easy though.

**Reply by jge162** [reply]
Click the plus (+) sign @cheese-lover

**Reply by cheese-lover** [reply]
It's showing my old repositories but I want to create another one, is there a limit though

**Reply by jge162** [reply]
Actually you cannot create repository on iphone or android on app.  I just checked my phone.

**Reply by jge162** [reply]
just have to go to the website github.com from web and login and create repository that way.

**Reply by jge162** [reply]
sorry not the answer you wanted. but it can be done from mobile broweser

**Reply by jge162** [reply]
was there anything else?

**Comment by Tmcniel** [comment]
Paid the fee to get my money and now it has kicked me out and is trying to make me do it all over again
